config/redis-helpers.js

```diff
@@ -0,0 +1,13 @@
+/* eslint no-process-env: "off" */
+
+const buildRedisUrl = () => {
```

Comment:
I'd be tempted to keep this within the config module as this breaks the rule of only declaring `process.env` within the config file (the eslint rule that you had to disable).

I know it might mean you won't be able to test it right now but I'd say put it in there because I'm doing a piece of work around Redis in #107 which will split building the redis client into a separate service where we can move this to and test it properly.

Reply:
Also, the check for production might not work fully as expected as if we run in Heroku, it will also expect the `REDIS_URL` as its set in the environment variables so it won't deploy there.

Reply:
@teneightfive I've rolled the helper method into `index.js`. I've exported it so that it can be tested - not sure whether I like that so let me know if you would rather not test that method at all (or do it a better way).

Also changed the check for cloud platform to the presence of `REDIS_AUTH_TOKEN` as you suggested.